.travis.yml

```diff
@@ -2,14 +2,11 @@ sudo: false
 language: node_js
 node_js:
   - "10"
-cache: npm
 
 services:
   - docker
 
 stages:
-  - cache
-  - eslint
```

Comment:
Should probably still run eslint.

`create-react-app` comes with a predefined preset that cannot be altered without ejecting. During the build, it only runs its version (and only on the react apps files).

The configuration file (https://github.com/apclary/dungeon-revealer/blob/master/.eslintrc.js) is ignored.

Reply:
See f5d53a7. Because of the postinstall script, things are running a bit out of order.

1. "build" stage "install:" implicitly runs `npm install`
    1. Install dependencies
    2. run postinstall script
        1. react eslint
        2. build binaries
2. "script:" runs `npm run eslint` using the repo [.eslintrc.js](https://github.com/apclary/dungeon-revealer/blob/master/.eslintrc.js)

So, the project is linted after it is built. However, this will still work as a check since the whole build will fail if any part of it fails.

Reply:
@n1ru4l do you think running eslint after the build is fine?

Reply:
@maxb2 yes, I even think that is totally fine!